core/drivers/atmel_tcb.c

```diff
@@ -50,7 +50,12 @@
 #define TCB_WPMR		0xe4
 #define  TCB_WPMR_WAKEY		0x54494d
 
+#ifdef CFG_DRIVERS_SAMA7G5_CLK
```

Comment:
Can't this be detected at runtime (from the DT perhaps?). Then you could avoid this new CFG.

Reply:
Thank you. `CFG_DRIVERS_SAMA7G5_CLK` has been updated to `CFG_SAMA7G5`.
Fixed others according to the comments and tags added to commit atmel_pio & atmel_rstc.

Reply:
Good enough for now, thanks.